onadata/apps/api/tests/viewsets/test_project_viewset.py

```diff
@@ -242,6 +242,37 @@ def test_projects_create(self):
             self.assertEqual(self.user, project.created_by)
             self.assertEqual(self.user, project.organization)
 
+    def test_create_duplicate_project(self):
+        """
+        Test creating a project with the same name
+        """
+        self._project_create()
```

Comment:
Please add comments to make it more clear that you have created two projects with the same name and that the second project creation fails with a validation error